recipes-bsp/u-boot/hab/env_resin.h

```diff
@@ -162,7 +161,7 @@
               "setexpr resin_roota ${resin_boot_part} + 1; " \
               "setexpr resin_rootb ${resin_boot_part} + 2; " \
               "run os_inc_bc_save;" \
-              "if test -n ${os_bc_lim}; then " \
+              "if test -n ${os_bc_lim} && test -n ${bootcount}; then " \
```

Comment:
This changes is also PRed into meta-balena in https://github.com/balena-os/meta-balena/pull/3560/commits/5fbf52822a0743ead7693364f41da7895d69ce72

Reply:
Unfortunately it's not an official NXP tool and the repository was i.MX8MM specific. I looked at other tools like https://github.com/usbarmory/crucible/tree/master which do support a broader range of chipsets, but apart from being in golang their UX was much worse.

In the end finding the OTP differences was just quicker.